examples/ecr/outputs.tf

```diff
@@ -5,17 +5,17 @@ output "repository" {
 
 output "registry_id" {
   description = "The registry ID where the repository was created."
-  value       = module.repository.repository.registry_id
+  value       = try(module.repository.repository.registry_id, null)
```

Comment:
this should not be needed as `.registry_id` is always available... what is the idea?

Reply:
same for the other `try()`s

Reply:
not if you have `module_enabled` set to `false`

Reply:
there is no module_enabled in there..

Reply:
It's used in the underlying module. See

https://github.com/mineiros-io/terraform-aws-ecr/blob/master/outputs.tf#L3 and
https://github.com/mineiros-io/terraform-aws-ecr/blob/master/main.tf#L15

Reply:
Won't terraform return an error on the outputs since they would return null in case we set `module_enabled = false` ?

Reply:
yes, but it is not set.... and it is also not settable...as the example module does not allow to set it... so it is not needed... ;)